### DISCUSSION TITLE
Is it possible to scale up/down StrimziPodSet using kubectl?

### DISCUSSION THREAD

**Discussion by djotanov** [strimzi/strimzi-kafka-operator#7254] [Q&A] [answered]
I have tried scaling using `kubectl scale --replicas=0 strimzipodset/my_cluster`, but I get error that resource my_cluster does not exist (even though it can be seen with kubectl get strimzistatefulset).

Is there any way to scale StrimziPodSet resources manually, with kubectl? This is needed when we want to shutdown our development clusters over the weekend, to save on compute costs (we usually scale down kafka, then zopokeepers down to 0, before shutting down the nodes, to avoid corruption).

**Comment by scholzj** [comment, marked as answer]
No, you cannot scale them down. This is not how they are designed, the do not have any number of replicas. You can delete them and later have the operator recreate them on Monday I guess.

**Reply by djotanov** [reply]
So I could scale down strimzi operator first, then delete StrimziPodSets.
What will happen with pods? Will they gracefully shutdown in this case, or there is potential for data corruption?

**Reply by scholzj** [reply]
The Pods will be deleted - it should not really differ from when you scale down a StatefulSet IMHO.
